pkgs/development/python-modules/liquidctl/default.nix

```diff
@@ -31,6 +32,13 @@ buildPythonPackage rec {
     hash = "sha256-LU8rQmXrEIoOBTTFotGvMeHqksYGrtNo2YSl2l2e/UI=";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/liquidctl/liquidctl/commit/c50afa4e610bd2e268e85c347e2644794c817a78.diff";
```

Comment:
```suggestion
      name = "tests-pillow-10.2.0-compat.patch";
      url = "https://github.com/liquidctl/liquidctl/commit/c50afa4e610bd2e268e85c347e2644794c817a78.diff";
```

Reply:
thanks, also applied this to the stable branch PR